gulpfile.js

```diff
@@ -1,5 +1,280 @@
+// README: Split this file up if it starts getting too long. ( i.e. 300 lines )
```

Comment:
I'll open an issue referencing this line about splitting this gulp file into configurable digestible chunks. cc: @theresaanna